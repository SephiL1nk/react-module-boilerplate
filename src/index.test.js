import React from 'react'
import List from './index.js'
import renderer from 'react-test-renderer'

describe('Snapshot testing and is everything rendering ', () => {
  test('Is snapshot the same', () => {
    const component = renderer.create(<List />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('Is hello world the text showing in front', () => {
    const component = renderer.create(<List />)
    const root = component.root
    const title = root.findByProps({ className: 'title' })

    expect(title.props.children).toBe('Hello world')
  })
})
