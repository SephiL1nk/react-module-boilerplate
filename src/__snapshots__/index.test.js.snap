// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing and is everything rendering  Is snapshot the same 1`] = `
<div
  className="title"
>
  Hello world
</div>
`;
